api/v1beta1/temporalcluster_types.go

```diff
@@ -351,6 +351,9 @@ type DatastoreSpec struct {
 	// TLS is an optional option to connect to the datastore using TLS.
 	// +optional
 	TLS *DatastoreTLSSpec `json:"tls,omitempty"`
+	// SkipCreate instructs the operator to skip creating the datastore.
```

Comment:
IMO `ship creating the datastore` is kind of confusing. 
```suggestion
	// SkipCreate instructs the operator to skip creating the database for SQL datastores or to skip creating keyspace for Cassandra. Use this option if your database or keyspace has already been provisioned by an administrator.
```

What do you think ?